core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -143,14 +143,22 @@ public boolean handle(int code) {
         StringHelper.randomId();
     }
 
+    public static void initializeProbes() {
```

Comment:
Can this be package private? I think its not needed except by bootstrap and bootstrap for testing.